torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2925,31 +2925,6 @@ at::Tensor XLANativeFunctions::upsample_nearest2d(
       GetOutputSizeWithScale(input_dims, scale_factors, output_size)));
 }
 
-at::Tensor XLANativeFunctions::upsample_nearest2d_backward(
```

Comment:
I assume we have `upsample_nearest2d_backward` backward test to make sure we don't regress?